pkg/cmd/roachtest/decommission.go

```diff
@@ -440,12 +441,18 @@ func runDecommissionAcceptance(ctx context.Context, t *test, c *cluster) {
 	}
 
 	t.l.Printf("trying to decommission a second time\n")
-	if buf, err := decommission(ctx, 2, c.Node(1), "decommission"); err != nil {
-		t.Fatalf("decommission failed: %v", err)
-	} else {
+	if err := testutils.SucceedsSoonError(func() error {
```

Comment:
But this will still fail hard on any branches that don't have your commit, no?

Reply:
How come is a _new_ roachtest applied on a branch with the _old_ code?

If there's such mix of version, what do you want me to do? You suggested a roachtest to check the warning, is that recommendation rescinded? I can do a TCL interactive test instead if needed.